### DISCUSSION TITLE
DockerTasks: DockerBuild - redirect stderr to stdout

### DISCUSSION THREAD

**Discussion by ITaluone** [nuke-build/nuke#1017] [Q&A] [answered]
Hi 

A quick question: is it possible to pass down the cli redirection symbol ">" to the DockerBuild task?

I want to acheive something like this:
`docker build . 2>&1`

Currently I am using this:

```csharp
Target DockerBuildImage => _ => _
        .Executes(() =>
        {
            DockerBuild(s => s
                .SetPath("...")
                .SetTag("XXXX")
                .SetQuiet(true));
        });
```

But the problem here is, that `--quiet` disables **all** of the output. 

So, long story short: I am looking for a way to keep the progress, but do not print it on stderr, because it is confusing and/or anoying to have lots of "ERR" in the build logs, but everything was fine.

**Comment by matkoch** [comment, marked as answer]
Addressed here: https://nuke.build/faq/

<img width="866" alt="Firefox-Frequently Asked Questions  NUKE-2022-09-16-bpa9un3T@2x" src="https://user-images.githubusercontent.com/5005566/190696073-91b8d945-0f50-4263-93ae-00d0dcdf7244.png">
